dice_roll.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+
+def roll_Dice():
+    print('Press R to roll the dice.\nPress Q if you want to quit this game.')
+    while True:
+        roll = input().upper()
+        if roll == 'R':
```

Comment:
What will happen if user enters something else than R or Q, there will be infinite while loop. We can terminate the program if an invalid input is entered.